test/Benchmarks/Metrics/ExemplarBenchmarks.cs

```diff
@@ -112,5 +132,18 @@ public void HistogramWithTagReduction()
 
             this.histogramWithTagReduction.Record(random.Next(1000), tags);
         }
+
+        public class LowValueExemplarFilter : ExemplarFilter
```

Comment:
want to use this is a to-be-written doc, to show the importance of using Filter to improve perf. (And show that Filter is the best way to improve perf, as it short circuits a lot of code, as opposed to using custom Reservoirs)